conf/messages.en

```diff
@@ -1016,7 +1044,9 @@ answerPage.section.trustsDetails.heading = Trust details
 answerPage.section.deceasedSettlor.heading = Settlor
 answerPage.section.settlors.heading = Settlors
 answerPage.section.settlor.subheading = Settlor {0}
+answerPage.section.leadTrustee.subheading = Lead trustee
 answerPage.section.trustees.heading = Trustees
+answerPage.section.leadTrusteeIndividual.heading = Lead Trustee
```

Comment:
Do we need both Lead Trustee header definitions?

Reply:
removed redundant message file entry "answerPage.section.leadTrusteeIndividual.heading"